### ISSUE TITLE
Check if entity is visible

### ISSUE THREAD

**Issue by SinanAkkoyun** [PrismarineJS/prismarine-entity#14]
I have to check if a given entity is visible.

My bot chases the invisible anti cheat fake players flying around him all the time.

I would love to see something like an entity.visible

Thank you!

**Comment by u9g** [comment]
could you just check the targetted player’s entity for status effects?

**Comment by SinanAkkoyun** [comment]
No, these fake players don't have an invisible potion on them...
They do have their invisible flag turned on. Thats another attribute

**Comment by SinanAkkoyun** [comment]
Or can somebody help me understand how to parse the metadata? I think thats it

**Comment by SinanAkkoyun** [comment]
Yesss, I did it, do a bitmask of x20 on the first entry  of metadata

Please add this as a simple entity.invisible, entity.onfire etc

**Comment by SinanAkkoyun** [comment]
console.log(player.entity.metadata[0] & 32 ? "yes" : "no")

**Comment by rom1504** [comment]
Feel free to open a pull request for this
